src/main/java/nl/novi/techiteasyhw/controller/CiModuleController.java

```diff
@@ -0,0 +1,54 @@
+package nl.novi.techiteasyhw.controller;
+
+import jakarta.validation.Valid;
+import nl.novi.techiteasyhw.dto.CiModule.CiModuleInputDto;
+import nl.novi.techiteasyhw.dto.CiModule.CiModuleOutputDto;
+import nl.novi.techiteasyhw.dto.Television.TelevisionOutputDto;
+import nl.novi.techiteasyhw.repository.CiModuleRepository;
+import nl.novi.techiteasyhw.service.CiModuleService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Objects;
+
+@RestController
+public class CiModuleController {
+
+    private final CiModuleService ciModuleService;
+
+    public CiModuleController (CiModuleService ciModuleService) {
+        this.ciModuleService = ciModuleService;
+    }
+
+    @PostMapping("/addcimodule")
+    public ResponseEntity <CiModuleOutputDto> addCiModule (@RequestBody CiModuleInputDto ciModuleInputDto) {
+        CiModuleOutputDto dto = ciModuleService.addCiModule(ciModuleInputDto);
+        return ResponseEntity.created(null).body(dto);
```

Comment:
Op de locatie van de null moet dus de URI komen waar we het in de chat al over hebben gehad